nsv13/code/modules/overmap/overmap_ghosts.dm

```diff
@@ -52,6 +52,19 @@
 	if(gunner)
 		QDEL_NULL(gunner)
 
+	//Buffs fighters so dogfights are more fun. 
```

Comment:
This needs to be modular based on the type of fighter, rather than hardcoded.

Reply:
Agree with karmic, try doing something like adding a spec_ghostship_changes() proc ../overmap that gets called when a ghost ship is created and have it called in place of your code, then have fighters override the proc to do what you want them to change.

Reply:
Done